cmd/argocd/commands/repo.go

```diff
@@ -171,7 +178,7 @@ func NewRepoAddCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 			fmt.Printf("repository '%s' added\n", createdRepo.Repo)
 		},
 	}
-	command.Flags().StringVar(&repo.Type, "type", common.DefaultRepoType, "type of the repository, \"git\" or \"helm\"")
+	command.Flags().StringVar(&repo.Type, "type", common.DefaultRepoType, "type of the repository, \"git\" or \"helm\" or \"helm-oci\"")
```

Comment:
I think it might be easier to keep only two types ( `git` and `helm`  ) and add helm specific flag that indicates `oci` repository. Otherwise, we need to update every `if else` statement related to repository type. E.g https://github.com/argoproj/argo-cd/blob/89a5bc87b9057d1a30a75ec0d75852379ff0ef26/util/db/helmrepository.go#L66